Traceback (most recent call last):
  File "/myriadfs/home/ucnwdma/Scratch/UCL_internship/./code/main.py", line 17, in <module>
    manager = StudyManager(f"./studies/{env.TIMESTAMP}/studies.db", f"./studies/{env.TIMESTAMP}/predictions.db") 
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/myriadfs/home/ucnwdma/Scratch/UCL_internship/code/study_manager.py", line 27, in __init__
    DatabaseManager(predictions_path)
  File "/myriadfs/home/ucnwdma/Scratch/UCL_internship/code/database_manager.py", line 27, in __init__
    self._create_tables()
  File "/myriadfs/home/ucnwdma/Scratch/UCL_internship/code/database_manager.py", line 32, in _create_tables
    with self._get_connection() as conn:
  File "/home/ucnwdma/miniconda3/envs/internship/lib/python3.11/contextlib.py", line 137, in __enter__
    return next(self.gen)
           ^^^^^^^^^^^^^^
  File "/myriadfs/home/ucnwdma/Scratch/UCL_internship/code/database_manager.py", line 67, in _get_connection
    conn.execute("PRAGMA journal_mode=WAL")  # Enable WAL mode for better concurrency
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
sqlite3.OperationalError: database is locked
